packages/core/src/hooks/useEthers.ts

```diff
@@ -64,14 +65,25 @@ export function useEthers(): Web3Ethers {
     isLoading,
   } = useNetwork()
 
-  const { networks } = useConfig()
+  const { networks, readOnlyUrls } = useConfig()
+  const error = useRef<Error | undefined>()
```

Comment:
Why `useRef`? Is it needed? Why not just `useState`?

Reply:
Ah yes, I forgot to change it